extensions/binding/org.eclipse.smarthome.binding.homematic/src/main/java/org/eclipse/smarthome/binding/homematic/internal/type/HomematicTypeGeneratorImpl.java

```diff
@@ -341,14 +342,15 @@ public ParameterOption createOption(String value, String description) {
                 }
             }
         }
-        
+
         configDescriptionProvider.addConfigDescription(new ConfigDescription(configDescriptionURI, parms, groups));
 
     }
 
     private URI getConfigDescriptionURI(HmDevice device) {
         try {
-            return new URI(String.format("%s:%s", CONFIG_DESCRIPTION_URI_THING_PREFIX, UidUtils.generateThingTypeUID(device)));
+            return new URI(
+                    String.format("%s:%s", CONFIG_DESCRIPTION_URI_THING_PREFIX, UidUtils.generateThingTypeUID(device)));
```

Comment:
how did this reformatting happen?

Reply:
Autoformatting in my newly installes Eclipse Photon IDE. I did not touch any settings...

Reply:
Once #5925 has been merged I give it a new try to setup my IDE because I want it to pick up the new warning settings regarding HTML files. For now I have fixed it by hand.